### PR TITLE
C2 API - Add and List downtimes

### PR DESCRIPTION
**Pull request by Guillaume28**
Now we have the capacity to add a realtime downtime with CLAPI commands for the hosts, services, hostgroups, servicegroups and instances.
In addition newman acceptance tests are included.

### CODE REVIEW

**Comment by lpinsivy** [comment]
@Guillaume28 dans tes fonction GetObjectByName or ByInstance, tu n'as pas de filtre pour sélectionner uniquement les ressources actives

**Comment by MatthieuMan** [comment]
Comments format are not as expected

File: tests/rest_api/rest_api.postman_collection.json [hunk 1]
```diff
@@ -1,8 +1,3 @@
-{
```
Comment (kduret) [inline tests/rest_api/rest_api.postman_collection.json]: Do not remove these lines

**Comment by ganoze** [comment]
@Guillaume28 also could you rebase against 2.8.x and resolve conflicts ?

File: tests/rest_api/rest_api.postman_collection.json [hunk 2]
```diff
@@ -47000,6 +47098,403 @@
...
+          "event": [
+            {
+              "listen": "prerequest",
+              "script": {
+                "type": "text/javascript",
+                "exec": [
+                  "postman.setGlobalVariable(\"fixed\", \"1\");",
+                  "",
+                  "var date = new Date();",
+                  "var datetime_start = date.getFullYear() + \"/\" + ('0' + (date.getMonth()+1)).slice(-2) + \"/\" + ('0' + date.getDate()).slice(-2) + \" \" + ('0' + dateEnd.getHours()).slice(-2) + \":\" + ",
```
Comment (loiclau) [inline tests/rest_api/rest_api.postman_collection.json]: make a function.

File: tests/rest_api/rest_api.postman_collection.json [hunk 3]
```diff
@@ -47000,6 +47098,403 @@
...
+              },
+              {
+                "key": "centreon-auth-token",
+                "value": "{{token}}",
+                "description": ""
+              }
+            ],
+            "body": {
+              "mode": "raw",
+              "raw": "{\n  \"action\": \"add\",\n  \"object\": \"RTDOWNTIME\",\n  \"values\": \"HOST;esx-alger-01;{{actual_start_time}};{{end_time}};1;3600;0;Faut pas renifler de la confiture sinon ça fait tousser\"\n}\n"
```
Comment (loiclau) [inline tests/rest_api/rest_api.postman_collection.json]: use a better comment

File: www/class/centreon-clapi/centreonRtDowntime.class.php [hunk 4]
```diff
@@ -45,13 +45,13 @@
...
 require_once "Centreon/Object/Relation/Downtime/Hostgroup.php";
 require_once "Centreon/Object/Relation/Downtime/Servicegroup.php";
-require_once realpath(dirname(__FILE__) . '/../centreonExternalCommand.class.php');
-require_once realpath(dirname(__FILE__) . '/../centreonDB.class.php');
-require_once realpath(dirname(__FILE__) . '/../centreonUser.class.php');
-require_once realpath(dirname(__FILE__) . '/../centreonGMT.class.php');
-require_once realpath(dirname(__FILE__) . '/../centreonHostgroups.class.php');
-require_once realpath(dirname(__FILE__) . '/../centreonServicegroups.class.php');
-require_once realpath(dirname(__FILE__) . '/../centreonInstance.class.php');
+require_once realpath(dirname(__FILE__).'/../centreonExternalCommand.class.php');
```
Comment (loiclau) [inline www/class/centreon-clapi/centreonRtDowntime.class.php]: style, add space

**Pull request closed without merging**